pydoctor/epydoc2stan.py

```diff
@@ -208,9 +208,12 @@ def from_epydoc(cls, field: EpydocField, source: model.Documentable) -> 'Field':
             arg=field.arg(),
             source=source,
             lineno=field.lineno,
-            body=field.body().to_stan(_EpydocLinker(source))
+            body=field.body()
             )
 
+    def format(self) -> Tag:
```

Comment:
I think that a docstring explaining the purpose of `format` would be nice :)

Reply:
I'll add one.